borgmatic/config/schema.yaml

```diff
@@ -19,8 +19,8 @@ properties:
         items:
             type: string
         description: |
-            List of source directories and files to backup. Globs and tildes are
-            expanded. Do not backslash spaces in path names.
+            List of source directories and files to back up. Globs and tildes
```

Comment:
This change is correct, but I'll admit I don't want it to be!

Reply:
Yeah, I was torn about this one… I just included it because "backing up" and "backed up" is used elsewhere.

Reply:
Totally makes sense.